R/error-helpers.R

```diff
@@ -646,3 +646,76 @@ abort_one_of_schema_table_names <- function() {
     .subclass = dm_error_full("one_of_schema_table_names")
   )
 }
+
+
+# write or read dm to/from files ------------------------------------------
+
+abort_dir_not_empty <- function() {
+  abort(
+    "Please choose a non-existent or empty directory for the csv-files.",
```

Comment:
Should we drop the dash here and elsewhere?

```suggestion
    "Please choose a non-existent or empty directory for the csv files.",
```